src/components/search/data/reducer.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/prefer-default-export */
+import {
+  DELETE_REFINEMENT, SET_REFINEMENT, CLEAR_REFINEMENTS, SET_REFINEMENTS_FROM_QUERY_PARAMS, ADD_TO_REFIENEMENT_ARRAY,
+  REMOVE_FROM_REFINEMENT_ARRAY,
+} from './actions';
+import { QUERY_PARAMS_TO_IGNORE } from './constants';
+
+export const valueArrayReducer = (state = [], action) => {
+  switch (action.type) {
+    case REMOVE_FROM_REFINEMENT_ARRAY:
+      return state.filter((value) => value !== action.value);
+    case ADD_TO_REFIENEMENT_ARRAY:
+      return [...state, action.value];
+    default:
+      return [...state];
+  }
+};
+
+export const refinementsReducer = (state = {}, action) => {
+  console.info('Refinement Action Triggered \n', action);
```

Comment:
remove console.log()

Reply:
So, if we suppress console.info in prod, I'd really like to keep this in, because it is super helpful for debugging.

Reply:
I'm compromising by commenting it out. ;)